task/04-arrays-tasks.js

```diff
@@ -321,81 +335,98 @@ function getPositivesCount(arr) {
  *   [ 'one','one','one','zero' ]     => [ 'zero','one','one','one' ]
  */
 function sortDigitNamesByNumericOrder(arr) {
-   throw new Error('Not implemented');
+  let numbers = [
+    "zero",
+    "one",
+    "two",
+    "three",
+    "four",
+    "five",
+    "six",
+    "seven",
+    "eight",
+    "nine"
+  ];
+  return arr.sort((a, b) => {
+    return numbers.indexOf(a) - numbers.indexOf(b);
+  });
 }
 
-/** 
+/**
  * Returns the sum of all items in the specified array of numbers
- * 
+ *
  * @param {array} arr
  * @return {number}
- * 
+ *
  * @example
  *   [] => 0
  *   [ 1, 2, 3 ]           => 6
  *   [ -1, 1, -1, 1 ]      => 0
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-   throw new Error('Not implemented');
+  return arr.reduce((sum, item) => sum + item, 0);
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.